devimint/src/tests.rs

```diff
@@ -2336,6 +2337,117 @@ pub async fn cannot_replay_tx_test(dev_fed: DevFed) -> Result<()> {
     Ok(())
 }
 
+pub async fn circular_deposit_test(dev_fed: DevFed) -> Result<()> {
```

Comment:
How much of an ask would it be to convert it a standalone devimint test like https://github.com/fedimint/fedimint/blob/master/modules/fedimint-meta-tests/src/bin/meta-sanity-test.rs ?

:D